### ISSUE TITLE
TalkBack doesn't read selected item on hover

### ISSUE THREAD

**Issue by crutchcorn** [oceanbit-dev/react-native-button-toggle-group#5]
As-of `v1.1.1` (with the new A11Y fixes), when using the swipe left/right methods of selecting items, we get the proper readout of the selected item:

![Expected behavior reading out as-intended](https://user-images.githubusercontent.com/9100169/106615924-cf3ade80-6521-11eb-9bfd-c37ea329e712.gif)

However, when trying to hover (tap) to select an item (not double-tap to select, just single tap to "read"), it simply "boops" to let you know that there's nothing there:

![Incorrect behavior](https://user-images.githubusercontent.com/9100169/106616206-26d94a00-6522-11eb-8fbd-143a7bc327a8.gif)

--------------

From a technical perspective, what's happening is that we are using a `MaskedView` to show the selected buttons over the unselected buttons. This allows us to do the trademark animation that we're using for the package.

However, despite using `accessibilityElementsHidden={true}`, Android refuses to read the button that's underneath the `MaskedView`, even when `pointerEvents` is set to `'none'` 

I tried for an hour to debug this but can't seem to figure out how to fix this issue.
